lib/Graph/Nodes.cpp

```diff
@@ -818,7 +818,9 @@ bool RowwiseQuantizedFullyConnectedNode::verify() const {
 
 bool GatherNode::verify() const {
   bool isValid = checkType(getResult(), getData().getElementType(), this);
-  isValid &= checkType(getIndices(), ElemKind::Int64ITy, this);
+  isValid &= checkType(
+      getIndices(),
+      llvm::makeArrayRef({ElemKind::Int64ITy, ElemKind::Int32ITy}), this);
```

Comment:
do you need explicitly "llvm::makeArrayRef" ?
What about just {ElemKind::Int64ITy, ElemKind::Int32ITy}?

Reply:
I just followed the convention in this file. I guess it doesn't matter too much here?

Reply:
Just harder to read with more things thrown here